REUSE.toml

```diff
@@ -4,7 +4,7 @@ SPDX-PackageSupplier = "The technology-data authors"
 SPDX-PackageDownloadLocation = "https://github.com/pypsa/technology-data"
 
 [[annotations]]
-path = ["uv.lock"]
+path = ["uv.lock", ".readthedocs.yaml", "mkdocs.yaml", "docs/*", "docs/*/*", "docs/*/*/*", "docs/*/*/*/*", ]
```

Comment:
Instead of adding all the files like this, please instead add the header to all files. Except for `uv.lock`, that should be possible.